pass: broken in Darwin

## Issue description

The new `pass` dependency `procps` apparently doesn't build on Darwin.

```
Undefined symbols for architecture x86_64:
  "_sched_setscheduler", referenced from:
      _main in test-schedbatch.o
ld: symbol(s) not found for architecture x86_64
clang-3.7: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [Makefile:429: test-schedbatch] Error 1
make[2]: Leaving directory '/private/var/folders/9b/v666t1m950qbvs8_jtd1mh680000gn/T/nix-build-procps-3.3.11.drv-0/procps-ng-3.3.11/testsuite'
make[1]: *** [Makefile:1468: all-recursive] Error 1
make[1]: Leaving directory '/private/var/folders/9b/v666t1m950qbvs8_jtd1mh680000gn/T/nix-build-procps-3.3.11.drv-0/procps-ng-3.3.11'
make: *** [Makefile:765: all] Error 2
builder for ‘/nix/store/97v57jlbmrdvb2qlgjycz29ifbq9rdp7-procps-3.3.11.drv’ failed with exit code 2
cannot build derivation ‘/nix/store/fkr1gy9wlv6gfckpraln61bmagwibphr-password-store-1.6.5.drv’: 1 dependencies couldn't be built
```

### Steps to reproduce

`nix-env -iA pkgs.pass`

## Technical details

* System: Mac OS X 10.11.6
* Nix version: nix-env (Nix) 1.11.4
* Nixpkgs version: 17.03pre91860.124f25b

--- a/pkgs/tools/security/pass/default.nix
+++ b/pkgs/tools/security/pass/default.nix
@@ -68,11 +68,11 @@ stdenv.mkDerivation rec {
     git
     gnupg
     gnused
-    procps
     pwgen
     tree
     which
-  ] ++ ifEnable x11Support [ dmenu xclip xdotool ]);
+  ] ++ stdenv.lib.optional stdenv.isLinux procps
+    ++ ifEnable x11Support [ dmenu xclip xdotool ]);
 
   postFixup = ''
     # Fix program name in --help

